test/TestModelRepository.sol

```diff
@@ -13,5 +13,15 @@ contract TestModelRepository {
 
     Assert.equal(repo.getNumModels(), initialModelsAmount, "ModelRepository holds 0 models initially");
   }
+  
+  function testIncentiveValue() {
+    ModelRepository repo = new ModelRepository();
+
+    //total error = initial error - target_error
+    //solved error = best_error - _new_model_error
+    //incentiveCalculate(bounty, total_error, solved_error)
 
+    Assert.equal(repo.incentiveCalculate(10000, 2000, 37), 185, "test case 1 ");
+    Assert.equal(repo.incentiveCalculate(123456789, 9, 4), 54869684, "test case 2");
```

Comment:
these two comments `test case 1/2` are unhelpful. maybe we can transfer the knowledge of line 20-22 into here
i would also do

```
bounty = 10000;
```

and the same for the other "magic numbers" to make it easier to read

Reply:
Okay. On it.